src/client/app/containers/app.ts

```diff
@@ -87,6 +90,10 @@ export class AppComponent {
             });
     }
 
+    get isViewMode() {
+        return /#\/view/.test(location.hash);
```

Comment:
If we refactored the existing code into a new component, we wouldn't need this location.hash test and it would make it easier to use history API.